pkgs/applications/networking/cluster/k3s/builder.nix

```diff
@@ -338,6 +338,7 @@ buildGoModule rec {
     util-linux # kubelet wants 'nsenter' from util-linux: https://github.com/kubernetes/kubernetes/issues/26093#issuecomment-705994388
     conntrack-tools
     runc
+    bash
```

Comment:
Do you know what exactly calls for bash?
There is also `runtimeShell`, if that can work, it may be preferable. For when using some other shell as default. Like in NixNG that seeks to use `busybox ash`.